src/main/java/core/basesyntax/model/Operation.java

```diff
@@ -0,0 +1,42 @@
+package core.basesyntax.model;
+
+public class Operation {
+    private final String type;
+    private final String fruit;
+    private final int amount;
+
+    public Operation(String type, String fruit, int amount) {
```

Comment:
```suggestion
    public Operation (Type type, String fruit, int amount) {
```
can we use type here?

Reply:
changed Operation to save Type instead of String